aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -46,6 +46,7 @@ static DimMask make_dim_mask(IntArrayRef dims, int64_t ndim) {
     mask.flip();
   } else {
     for (int64_t dim : dims) {
+      TORCH_CHECK(dim < 64, "PyTorch doesn't support reduction operations for dim>=64");
```

Comment:
It would be good to refactor this 64 and the 64 here: https://github.com/pytorch/pytorch/blob/9d9bc93bfba9328a419e533cd2c74b831f60bd61/aten/src/ATen/native/TensorIterator.h#L143 into the same `constexpr int64_t`.

Also as Greg mentioned, we should check that the output of `maybe_wrap_dim(dim, ndim)` is less than 64.